runner-config.md

```diff
@@ -0,0 +1,44 @@
+To run the tests with a customized configuration, you can add a new setting with key named: `java.test.config`.
```

Comment:
Edit:
You can run tests with custom configuration. The configuration can be specified in `settings.json` using the key `java.test.config`.